.github/workflows/.test.yml

```diff
@@ -142,7 +142,6 @@ jobs:
           buildkitd-flags: --debug
       -
         name: Test
-        continue-on-error: ${{ matrix.tags == 'nydus' }}
```

Comment:
Was this just for testing? I think we should still keep this? cc @crazy-max

Reply:
I think we can remove it if we don't expect flaky tests with nydus anymore.

Reply:
Let's remove it to ensure nydus tests passed. :)